### PR TITLE
Add loganalyzer ignore regex for GCU

### PR DESCRIPTION
**Pull request by wen587**
### Description of PR

Summary: Add LogAnalyzer ignored regex for GCU as it will report failure on incorrect ordering which is discarded by GCU
Fixes # (issue)
```
admin@vlab-01:~/loganalyzer$ sudo config apply-patch rm0.json
Patch Applier: Patch application starting.
Patch Applier: Patch: [{"op": "remove", "path": "/LOOPBACK_INTERFACE"}]
...

**(below sorting error will be treated as error by LogAnalyzer.)**     <===============

libyang[0]: Leafref "../../LOOPBACK_INTERFACE_LIST/name" of value "Loopback0" points to a non-existing leaf. (path: /sonic-loopback-interface:sonic-loopback-interface/LOOPBACK_INTERFACE/LOOPBACK_INTERFACE_IPPREFIX_LIST[name='Loopback0'][ip-prefix='FC00:1::32/128']/name)
sonic_yang(3):Data Loading Failed:Leafref "../../LOOPBACK_INTERFACE_LIST/name" of value "Loopback0" points to a non-existing leaf.
...
Patch Applier: The patch was sorted into 3 changes:
Patch Applier:   * [{"op": "remove", "path": "/LOOPBACK_INTERFACE/Loopback0|10.1.0.32~132"}]
Patch Applier:   * [{"op": "remove", "path": "/LOOPBACK_INTERFACE/Loopback0|FC00:1::32~1128"}]
Patch Applier:   * [{"op": "remove", "path": "/LOOPBACK_INTERFACE"}]
Patch Applier: Applying 3 changes in order:
Patch Applier:   * [{"op": "remove", "path": "/LOOPBACK_INTERFACE/Loopback0|10.1.0.32~132"}]
Patch Applier:   * [{"op": "remove", "path": "/LOOPBACK_INTERFACE/Loopback0|FC00:1::32~1128"}]
Patch Applier:   * [{"op": "remove", "path": "/LOOPBACK_INTERFACE"}]
Patch Applier: Verifying patch updates are reflected on ConfigDB.
Patch Applier: Patch application completed.
```
### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012

#### What is the motivation for this PR?
Recent LogAnalyzer fix report failure on discarded ordering in GCU jsonpatch. However, GCU doesn't fail actually as it will explore a correct ordering and apply.
#### How did you do it?
Disable LogAnalyzer for GCU tests.
#### How did you verify/test it?
Run GCU test without flag --disable_loganalyzer and see if it works.

### CODE REVIEW

**Review by ghooo — DISMISSED** [review]

**Review by wangxin — CHANGES_REQUESTED** [review]
It is not a good practice to totally disable loganalyzer. Real issue could be hide using this method. It is better to add the message pattern to ignore list.

**Comment by wen587** [comment]
> It is not a good practice to totally disable loganalyzer. Real issue could be hide using this method. It is better to add the message pattern to ignore list.

Changed to ignore regex. Will fix the failure in furture PR. Currently add the ignored_regex for PR check pass.

File: tests/generic_config_updater/conftest.py [hunk 1]
```diff
@@ -2,6 +2,35 @@
...
+             ".*ERR.*Failed to start dhcp_relay container.*", # test_dhcp_relay
+             ".*ERR GenericConfigUpdater: Service Validator: Service has been reset.*", # test_dhcp_relay test_syslog
+             ".*Same listen range is attached to peer-group.*", # test_bgp_speaker -> real issue
+             ".*ERR swss[0-9]*#orchagent.*removeLag.*", # autorestart/test_container_autorestart.py test_portchannel_interface
+             ".*ERR swss[0-9]*#intfmgrd: :- setIntfVrf:.*", # test_portchannel_interface
+             ".*ERR teamd.*get_dump: Can't get dump for LAG.*", # test_portchannel_interface
+             ".*ERR kernel.*Reset adapter.*", # test_portchannel_interface replace mtu
+             ".*ERR swss[0-9]*#orchagent: :- getPortOperSpeed.*", # test_portchannel_interface replace mtu
+             ".*ERR.*Failed to apply Json change.*", # validator need updater submodule
+             ".*ERR GenericConfigUpdater: Change Applier: service invoked.*", # validator need updater submodule
```
Comment (wangxin) [inline tests/generic_config_updater/conftest.py:30]: Are all these errors only be triggered during GCU testing?
If these errors can be observed in other tests, they need to be added to the common ignore list.

Comment (wen587) [reply tests/generic_config_updater/conftest.py:30]: These error are only for GCU test.

**Review by wangxin — APPROVED** [review]

**Comment by gord1306** [comment]
@wen587 @wangxin, is it better to put all these error log to the expected error messages instead of ignore messages ??

**Pull request merged**